src/SeleniumLibrary/keywords/waiting.py

```diff
@@ -25,20 +25,17 @@ class WaitingKeywords(LibraryComponent):
 
     @keyword
     def wait_for_condition(self, condition, timeout=None, error=None):
```

Comment:
There has been request to improve the documentation with examples https://github.com/robotframework/SeleniumLibrary/issues/188 Should we provide few examples in this PR?

Reply:
I'd do it separately. A note about that could be added to #925.

Reply:
On